lib/haml_lint/tree/tag_node.rb

```diff
@@ -198,5 +206,17 @@ def script
     def text
       (@value[:value] if @value[:parse]) || ''
     end
+
+    private
+
+    def parsed_attributes
+      HamlLint::RubyParser.new.parse(hash_attributes_source)
```

Comment:
Ah, I forgot we were in the `TagNode`. Sounds like we should define a `parse_ruby` helper on the `Node` class, since I see this being something being more heavily used in node helpers going forward.